substrate/frame/revive/mock-network/src/parachain/contracts_config.rs

```diff
@@ -15,13 +15,22 @@
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
 use super::{Balances, Runtime, RuntimeCall, RuntimeEvent};
-use crate::parachain::RuntimeHoldReason;
+use crate::parachain::{parameter_types, MaxInstructions, RuntimeHoldReason};
 use frame_support::derive_impl;
+use pallet_revive::precompiles::builtin::xcm::XcmPrecompile;
+use xcm::latest::Weight;
+use xcm_builder::FixedWeightBounds;
+
+parameter_types! {
+	pub const BaseXcmWeight: xcm::latest::Weight = Weight::from_parts(100_000_000, 2_000);
+}
 
 #[derive_impl(pallet_revive::config_preludes::TestDefaultConfig)]
 impl pallet_revive::Config for Runtime {
 	type AddressMapper = pallet_revive::AccountId32Mapper<Self>;
 	type Currency = Balances;
 	type Time = super::Timestamp;
 	type Xcm = pallet_xcm::Pallet<Self>;
+	type Precompiles = (XcmPrecompile<Self>,);
```

Comment:
Just add it to the `Production` type alias. It is built in and always available until moved out of the pallet. No need to add it here.